pkg/cluster/cert.go

```diff
@@ -171,10 +169,6 @@ func GetSecret(ctx context.Context, c client.Client, namespace, name string) (*v
 }
 
 func copySecretToNamespace(ctx context.Context, c client.Client, secret *v1.Secret, newSecretName, namespace string) error {
-	// Get default name if newSecretName is empty
-	if newSecretName == "" {
-		newSecretName = DefaultCertificateSecretName
-	}
```

Comment:
I am not sure we need to default this to anything. I would assume the caller does due diligence.

Reply:
If line 142 is the only usage, I think it is always going to be populated...